spec/support/fixtures/bookshelf/config/environment.rb

```diff
@@ -24,10 +24,11 @@ module Admin
 
     Assets = Hanami::Assets.duplicate(self) do
       root             "spec/support/fixtures/bookshelf/apps/admin"
-      public_directory "tmp/bookshelf/public"
+      public_directory "spec/tmp/bookshelf/public"
```

Comment:
@landongrindheim This is wrong, as per effect of `TMP` referencing a wrong directory. The correct value should be `<root-of-the-git-repository>/tmp`.

Reply:
I'll update `TMP`'s path and return the configs to their original state 😄